mne/io/edf/edf.py

```diff
@@ -1022,7 +1016,12 @@ def _check_stim_channel(stim_channel, ch_names, sel):
 
     elif isinstance(stim_channel, str):
         if stim_channel == 'auto':
-            valid_stim_ch_names = DEFAULT_STIM_CH_NAMES
+            if 'auto' in ch_names:
+                warn(RuntimeWarning, 'Using `stim_channel=\'auto\'` when auto'
```

Comment:
You could use a raw string here to avoid the backslashes. Also, the comma in the first sentence should be removed, and there should be a period after the last sentence.